core/src/main/java/org/bitcoinj/script/ScriptBuilder.java

```diff
@@ -438,6 +438,22 @@ public static Script updateScriptWithSignature(Script scriptSig, byte[] signatur
         return builder.build();
     }
 
+    /**
+     * Creates a segwit scriptPubKey that sends to the given public key hash.
+     */
+    public static Script createP2WPKHOutputScript(byte[] hash) {
+        checkArgument(hash.length == 20);
```

Comment:
I believe we could use the constant `SegwitAddress.WITNESS_PROGRAM_LENGTH_PKH` here.

Reply:
Right